### PR TITLE
[FX-2286] Implements show2 click tracking

### PR DESCRIPTION
**Pull request by dzucconi**
Re: [FX-2286](https://artsyproduct.atlassian.net/browse/FX-2286)

This takes care of: `clickedViewingRoomCard`,`clickedFairCard`, and `clickedPartnerCard`

Is there anything that has to happen for the artwork grid events or are those just included with the component? Ditto re: `pageView`?

### CODE REVIEW

**Comment by sweir27** [comment]
Artwork grid tracking should be okay, but I would verify.

Re: pageviews, it's possible that they're happening _twice_ (can also be verified in the console). I think we'll need to update [this file](https://github.com/artsy/force/blob/17c3aa9384cbfc8c557b3ed0fb04f6ee661ba1bc/src/desktop/assets/analytics.ts#L8-L28) (but will have to make sure to not silence _existing_ page views until we've launched).

**Comment by mzikherman** [comment]
Nice, we can always incrementally add more analytics.

I think for artwork grid tracking, that does need to be passed in to the component via `onArtworkBrickClick` from the calling code.

File: src/v2/Apps/Show/Components/ShowContextCard.tsx [hunk 1]
```diff
@@ -34,14 +37,34 @@ const CARD_IMAGE_WIDTHS = [
...
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
   const FairInfo = () => {
+    const handleClick = () => {
+      tracking.trackEvent({
+        context_module: ContextModule.presentingFair,
```
Comment (eessex) [inline src/v2/Apps/Show/Components/ShowContextCard.tsx:50]: Are these events not yet in cohesion? there is no action type here!

Comment (dzucconi) [reply src/v2/Apps/Show/Components/ShowContextCard.tsx:50]: I'm not sure I follow?

**Pull request merged**